prow/cluster/jobs/istio/cni/istio.cni.master.yaml

```diff
@@ -113,6 +113,54 @@ postsubmits:
           privileged: true
       nodeSelector:
         testing: test-pool
+  - annotations:
+      testgrid-alert-email: istio-oncall@googlegroups.com
+      testgrid-dashboards: istio-cni-postsubmits-master
+      testgrid-num-failures-to-alert: "1"
+    branches:
+    - ^master$
+    decorate: true
+    extra_refs:
+    - base_ref: master
+      org: istio
+      path_alias: istio.io/istio
+      repo: istio
+    name: cni-e2e-master
+    path_alias: istio.io/cni
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - make
+        - prow-e2e
+        image: gcr.io/istio-testing/istio-builder:v20190807-7d818206
+        name: ""
+        resources:
+          limits:
+            cpu: "3"
+            memory: 24Gi
+          requests:
+            cpu: 500m
+            memory: 3Gi
+        securityContext:
+          privileged: true
```

Comment:
Would be nice if we can figure out how to do these builds without privilege at some point, like by using kaniko

Reply:
can we run kind without privilege? if possible I would love to but I didn't think that was feasible


for other tests though like unit tests we should definitely